language/stdlib/modules/LibraAccount.move

```diff
@@ -935,28 +863,32 @@ module LibraAccount {
             txn_sequence_number == sender_account.sequence_number,
             PROLOGUE_ESEQUENCE_NUMBER_TOO_NEW
         );
-        assert(
-            LibraTimestamp::now_seconds() < txn_expiration_time_seconds,
-            PROLOGUE_ETRANSACTION_EXPIRED
-        );
     }
 
     /// Collects gas and bumps the sequence number for executing a transaction
-    fun epilogue<Token>(
-        sender: address,
+    /// This function can only be called during the epilogue as any invokation during the execution body will cause
+    /// the sequence number to be poisoned.
+    public fun charge_transaction<Token>(
```

Comment:
Note that although this function is public, it is not invokable from regular user scripts as it will cause the sequence number to be incremented and thus fail the assertion at line 885.